dhall-toml/dhall-toml.cabal

```diff
@@ -0,0 +1,78 @@
+Name:               dhall-toml
+Version:            0.1.0
+Cabal-Version:      >=1.10
+Build-Type:         Simple
+Tested-With:        GHC == 8.4.3, GHC == 8.6.1
```

Comment:
```suggestion
```

AFAIK we currently only have CI with GHC 8.6 and GHCJS. We're not doing a good job updating the `Tested-With` stanzas though, so I suggest simply not recording this info at all.